incubator/hnc/Makefile

```diff
@@ -30,17 +39,38 @@ test: build
 # Builds all binaries (manager and kubectl) and manifests
 build: generate fmt vet manifests
 	go build -o bin/manager ./cmd/manager/main.go
-	go build -o bin/kubectl-hnc ./cmd/kubectl/main.go
+	go build -o bin/kubectl/kubectl-hnc ./cmd/kubectl/main.go
 
+# Clean all binaries (manager and kubectl)
+clean:
+	rm -r bin/kubectl/kubectl-hnc
+	rm -f bin/manager
+	
 # Install kubectl plugin
 kubectl: build
-	go build -o ${GOPATH}/bin/kubectl-hnc ./cmd/kubectl/main.go
```

Comment:
Any reason need to rebuild binary rather than copy it? I see no purpose to initiate build here.  Therefore i replace it.

Reply:
sgtm